aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -1065,24 +1065,6 @@ Tensor triton_scaled_dot_attention(const Tensor& q, const Tensor& k, const Tenso
 
 REGISTER_CUDA_DISPATCH(_fused_sdp_choice_stub, &_fused_sdp_choice_cuda);
 
-// !!This function is deprecated. See FunctionsManual.cpp for the implementation!!
```

Comment:
It might be better to do the deletion of this function and the addition of the batch rule in separate PRs so that it's easier to review. But totally up to you.